statsmodels/robust/scale.py

```diff
@@ -14,7 +14,8 @@
 from . import norms
 from statsmodels.tools import tools
 
-def mad(a, c=Gaussian.ppf(3/4.), axis=0, center=np.median):
+
+def mad(a, c=None, axis=0, center=np.median):
```

Comment:
Gaussian.ppf(3/4.) is a float scalar and not mutable
?

Reply:
@bashtage josef is right on this one.  If reverting that will get this otherwise-solid PR through, go for it.